x-pack/plugins/lens/public/datasources/form_based/operations/definitions/terms/index.tsx

```diff
@@ -889,7 +945,10 @@ The top values of a specified field ranked by the chosen metric.
             aria-label={i18n.translate('xpack.lens.indexPattern.terms.orderDirection', {
               defaultMessage: 'Rank direction',
             })}
-            isDisabled={currentColumn.params.orderBy.type === 'rare'}
+            isDisabled={
+              currentColumn.params.orderBy.type === 'rare' ||
```

Comment:
nit: this can be computed only once `isRareOrSignificant` and reused here and below.